mypy/stubgenc.py

```diff
@@ -379,13 +385,25 @@ def generate_c_function_stub(
             # a sig generator indicates @classmethod by specifying the cls arg
             if class_name and signature.args and signature.args[0].name == "cls":
                 output.append("@classmethod")
-            output.append(
-                "def {function}({args}) -> {ret}: ...".format(
-                    function=name,
-                    args=", ".join(args),
-                    ret=strip_or_import(signature.ret_type, module, known_modules, imports),
+            if include_docstrings and docstr:
+                output.append(
+                    "def {function}({args}) -> {ret}:".format(
+                        function=name,
+                        args=", ".join(args),
+                        ret=strip_or_import(signature.ret_type, module, known_modules, imports),
+                    )
+                )
+                docstr_quoted = mypy.util.quote_docstring(docstr.strip())
+                docstr_indented = "\n    ".join(docstr_quoted.split("\n"))
+                output.extend(f"    {docstr_indented}".split("\n"))
+            else:
+                output.append(
+                    "def {function}({args}) -> {ret}: ...".format(
+                        function=name,
+                        args=", ".join(args),
+                        ret=strip_or_import(signature.ret_type, module, known_modules, imports),
+                    )
```

Comment:
Isn't this part completely repeats the other branch? If yes, then please use
```python
if foo:
   bar
baz
```
instead of
```python
if foo:
    bar baz
else:
    baz
```

Reply:
To be more precise, in this case you need:
```pyton
bar
if foo:
    baz
```
since you always need a type, and docstring should come after (only if the flag is enabled)